src/electron/main/main.ts

```diff
@@ -11,7 +11,7 @@ const createWindow = () => {
         webPreferences: { nodeIntegration: true },
         frame: false,
         width: 600,
-        height: 391,
+        height: 500,
```

Comment:
500 seems a little tall for the window. Is there a way we could make the telemetry popup wider rather than make the window taller? That would look closer to the figma. Feel free to ignore this if you're already planning on tuning the style in a future PR.

Reply:
That's a good point... I'd look into how much work is it to make it wider without affecting the size on the extension. If it's easy, I'll do it as part of this PR, if it's not, I'll have a second PR to fix this.

Reply:
@jalkire I was able to override the dialog size without modifying the windows size or affecting how the dialog looks like on the extension. Thanks for the feedback